lib/blackout_generator.rb

```diff
@@ -0,0 +1,29 @@
+module BlackoutGenerator
+  def generate
+    Blackout.create! do |blk|
+      blk.start_date = time_rand(Time.zone.now, Time.zone.now + 1.year)
+      blk.end_date = time_rand(blk.start_date, blk.start_date.next_week)
+      blk.notice = FFaker::HipsterIpsum.paragraph(2)
+      blk.created_by = User.first.id
+      blk.blackout_type = %w(soft hard).sample
+    end
+  end
+
+  private
+
+  def time_rand(start, finish)
+    range = (start..finish)
+    random_time = rand(range)
+    blackouts = Blackout.all.map { |blk| blk.start_date..blk.end_date }
```

Comment:
Yea, I don't see a reason to ensure that our blackouts don't overlap, we don't validate for that in real life so let's not do it here. We can basically just remove all the private methods here and just use `rand` to generate the start and end dates. What do you think?